wkcuber/downsampling.py

```diff
@@ -90,10 +95,14 @@ def create_parser():
     )
 
     parser.add_argument(
-        "--compress", action="store_true", help="Compress data during downsampling"
+        "--compress",
+        action="store_true",
+        help="Compress data during downsampling",
+        default=True,
```

Comment:
how do I deactivate compression if `store_true` and `default=True`?

Reply:
Very valid point. I inverted the argument (the same as the main script now).